openapi/v1.1.0/optimade.json

```diff
@@ -301,7 +301,7 @@
             "required": false,
             "schema": {
               "title": "Page Number",
-              "minimum": 0.0,
+              "minimum": 1.0,
```

Comment:
Is it OK for validation to fail because of RECOMMENDED type violation? Just asking, sadly I do not recall what was the agreement regarding the degrees of insistence.

Reply:
Could you clarify? Are you referring to this page-number change in particular?

Reply:
Also the `default` should not be 0 then.

Reply:
Prior to this PR, `minimum` value for `page_number` was 0. Now the PR makes it 1. This means usage of `page_number=0` will fail validation, but the specification does not forbid (just recommends against) such value. Thus it seems to me that the schemas for validation are now stricter then the specification.

Reply:
Ah, I'm with you now. The spec always takes precedence, so of course you can argue that using 0 would be fine. We just have to use 1 to encode this constraint on our server. Thanks for pointing out the default @blokhin, we'll fix that.

Reply:
If we do not define whether page based pagination is 0 or 1 based, we cannot use 0 or 1 as the default value. Would "None", indicating that the value of the parameter has not been set, be a good alternative?

Reply:
@ml-evs I find it a bit uncomfortable that the schemas in this repository are intentionally not conforming to the word of the specification. AFAIU you are using the schemas for your server as they are, but I tend to think reflecting specification as closely as possible is important. Maybe for your server you could fork+modify the repository, or keep a server-specific patch?

Reply:
Understood, hence why I haven't been pushing too hard to merge this (happy to put it on hold for now). It really depends what the intention for the schemas hosted here are (see #1!) - either way, this PR now updates the minimum value from an arbitrary value (0) to the value "RECOMMENDED" by the specification (1).

Reply:
I take that "RECOMMENDED" means that an implementation not adhering to this requirement (i.e., having pagination starting from 0) is still conforming to OPTIMADE. However, this OpenAPI schema imposes a strict requirement of page numbers starting from 1.

A scenario: Let's take a tool generating nice GUIs straight from OpenAPI schemas (I do not have a link for such a tool, though). Such tools would generate GUIs which would not allow entering page number of 0, rendering first pages in some of the implementations (conforming to OPTIMADE) inaccessible.

My opinion is that these OpenAPI schemas must not be stricter than the OPTIMADE specification. Even given the ongoing discussion in #1 I believe such property should be retained.